test/functionalTest/harnessFunctions.sh

```diff
@@ -1129,6 +1148,100 @@ function dbInsertEntity()
 }
 
 
+
+# ------------------------------------------------------------------------------
+#
+# kafkaCreateTopics -
+#
+# Create one or more topics in Kafka
+kafkaCreateTopics() {
+  if [ "$#" -lt 2 ]; then
+    echo "Uso: kafkaCreateTopics <bootstrap> <topic1> [topic2 ...]"
+    echo "Ej.:  kafkaCreateTopics kafka1:9092 sub1 sub2"
+    return 2
+  fi
+
+  local bootstrap="$1"; shift
+
+  for topic in "$@"; do
+    echo "Creating topic: $topic (bootstrap: $bootstrap)"
+
+    /opt/kafka/bin/kafka-topics.sh \
```

Comment:
We should avoid absolute pathnames here. In the same why why use for instance `mongosh` assuming that is is within the execution paths.

Probably needs also to adjust the documentation about installing the client. I'll have a look.